Ds3/Calls/DeleteDataPersistenceRuleSpectraS3Request.cs

```diff
@@ -47,7 +54,7 @@ internal override string Path
         {
             get
             {
-                return "/_rest_/data_persistence_rule/" + DataPersistenceRule;
+                return "/_rest_/data_persistence_rule/" + DataPersistenceRuleId.ToString();
```

Comment:
Ok.  Let's update it in another pull request to remove it since it is unnecessary.